mist-api/src/routes/genomes/get.js

```diff
@@ -22,17 +23,54 @@ module.exports = function(app, middlewares, routeMiddlewares) {
 				'taxonomy_id',
 			],
 		}),
-		// Provide for searching against name
+		// Provide for searching against name, taxonomy levels and assembly_level
+		//In a query all the terms except 'search' and 'assembly_level' are expected to be one word terms, nevetherless as a precaution
```

Comment:
I'm a stickler for consistent style. When writing comments, please add space after `//`.